### PR TITLE
Extracted l10n messages from 2023-10-04 at 3122b60698

### PR DESCRIPTION
**Pull request by diox**

### CODE REVIEW

File: locale/templates/LC_MESSAGES/django.pot [hunk 1]
```diff
@@ -4091,6 +4083,27 @@ msgstr ""
 msgid "More information on Developer Accounts"
 msgstr ""
 
+#: src/olympia/devhub/templates/devhub/includes/android_compatibility_modal.html:3
+msgid "Firefox Android compatibility"
```
Comment (flodolo) [inline locale/templates/LC_MESSAGES/django.pot:4087]: The name of the app is Firefox for Android, not Firefox Android. Same note applies to the following two strings.

File: locale/templates/LC_MESSAGES/django.pot [hunk 2]
```diff
@@ -4091,6 +4083,27 @@ msgstr ""
...
+"extensions-for-firefox-for-android/\">Developing extensions for Firefox "
+"Android</a> for test guidance before submitting."
+msgstr ""
+
+#: src/olympia/devhub/templates/devhub/includes/android_compatibility_modal.html:6
+msgid "Yes, I’ve tested my Firefox Android extension"
+msgstr ""
+
+#: src/olympia/devhub/templates/devhub/includes/android_compatibility_modal.html:7
+msgid "No, I have not tested for Android compatibility"
```
Comment (flodolo) [inline locale/templates/LC_MESSAGES/django.pot:4104]: Why did we switch to "Android" directly, when we're actually talking about compatibility with Firefox for Android?

File: locale/templates/LC_MESSAGES/django.pot [hunk 3]
```diff
@@ -7129,58 +7142,86 @@ msgid ""
...
+msgid "Cookies"
+msgstr ""
+
+#: src/olympia/templates/photon-footer.html:54
+msgid "Legal"
+msgstr ""
+
+#: src/olympia/templates/photon-footer.html:57
+msgid ""
+"Except where otherwise <a href=\"https://www.mozilla.org/en-"
```
Comment (flodolo) [inline locale/templates/LC_MESSAGES/django.pot:7221]: Never put `en-US` in links to mozilla.org or SUMO, let the website manage the locale negotiation.
https://www.mozilla.org/about/legal/

**Review by flodolo — COMMENTED** [review]
CC @peiying2 (I ended up commenting her, you can ignore comments in Pontoon)

File: locale/templates/LC_MESSAGES/django.pot (hunk 3 above)
Comment (diox) [reply locale/templates/LC_MESSAGES/django.pot:7221]: :+1: I just happened to notice this footer wasn't translated but I didn't double-check the links when adding the gettext tag...

**Pull request merged**